src/sbt-test/settings/lazy-protoc-resolution/build.sbt

```diff
@@ -1 +1,20 @@
-PB.protocDependency := "non" % "existing" % "artifact"
+// PB.targets, no proto
```

Comment:
that's most likely the scenario from https://github.com/thesamet/sbt-protoc/issues/304